scripts/get_baseline_hash.py

```diff
@@ -21,7 +22,8 @@ def filter_results(issue):
         if label['name'] in filter_labels:
             labels_check = False
             break
-    title_check = "Testsuite Status" in issue["title"]
+    regex_check = re.search("^Testsuite Status.*$", issue["title"]) # if fails, regex_check == None
```

Comment:
I think we can drop the `.*$` and get the same behavior

Reply:
just double checked and yeah we can